lib/implore.es6.js

```diff
@@ -60,8 +61,22 @@ export default function implore(request) {
 			'implore requires parameter `method` to be a string'
 		);
 
+		const {requestHeaders={}} = request;
+		invariant(
+			typeof requestHeaders === 'object',
+			'implore requires parameter `requestHeaders` to be an object'
+		);
+
```

Comment:
The documentation will be clear what we expect. I'm not worried about checking everything, just basic checks. For example, we don't check empty strings, just things are strings. If you want to try and break the code then go for it.